src/instructlab/model/serve.py

```diff
@@ -20,17 +19,23 @@
 @click.command()
 @click.option(
     "--model-path",
-    type=click.Path(),
-    default=config.DEFAULT_MODEL_PATH,
-    show_default=True,
+    cls=clickext.ConfigOption,
+    type=click.Path(path_type=pathlib.Path),
+    required=True,  # default from config
     help="Path to the model used during generation.",
 )
 @click.option(
     "--gpu-layers",
+    # cls=ConfigOption,
```

Comment:
left over?

Reply:
No, not implemented yet. The default comes from `llama_cpp.gpu_layers`, not `gpu_layers`.